src/style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* body {
+  width: 96px;
+  height: 96px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: #fff;
+} */
+
+body {
+  --basic-size: 16px;
+}
+
+.stars__row {
+  width: 96px;
+  display: flex;
+}
+
+.stars__star {
+  width: var(--basic-size);
+  height: var(--basic-size);
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin: 0 2px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

Comment:
You could simplify it.
Create a selector using :last-child and :not pseudo classes and apply `margin-right: 4px;`